packages/cactus-plugin-persistence-fabric/package.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "@hyperledger/cactus-plugin-persistence-fabric",
+  "version": "1.1.3",
+  "description": "A web service plugin that provides management capabilities on a Cactus Plugin Fabric Block Persistance as a scrapper of blocks.",
+  "keywords": [
+    "Hyperledger",
+    "Cactus",
+    "Integration",
+    "Blockchain",
+    "Distributed Ledger Technology"
+  ],
+  "homepage": "https://github.com/hyperledger/cactus#readme",
+  "bugs": {
+    "url": "https://github.com/hyperledger/cactus/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger/cactus.git"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Hyperledger Cactus Contributors",
+    "email": "cactus@lists.hyperledger.org",
+    "url": "https://www.hyperledger.org/use/cactus"
+  },
+  "contributors": [
+    {
+      "name": "Barnaba Pawelczak",
+      "email": "barnaba.pawelczak@fujitsu.com",
+      "url": "https://fujitsu.com"
+    },
+    {
+      "name": "Michal Bajer",
+      "email": "michal.bajer@fujitsu.com",
+      "url": "https://fujitsu.com"
+    },
+    {
+      "name": "Eryk Baranowski",
+      "email": "Eryk.Baranowski@fujitsu.com",
+      "url": "https://fujitsu.com"
+    },
+    {
+      "name": "Tomasz Awramski",
+      "email": "tomasz.awramski@fujitsu.com",
+      "url": "https://fujitsu.com"
+    }
+  ],
+  "main": "dist/lib/main/typescript/index.js",
+  "module": "dist/lib/main/typescript/index.js",
+  "browser": "",
+  "types": "dist/lib/main/typescript/index.d.ts",
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "codegen": "run-p 'codegen:*'",
+    "codegen:openapi": "npm run generate-sdk",
+    "generate-sdk": "openapi-generator-cli generate -i ./src/main/json/openapi.json -g typescript-axios -o ./src/main/typescript/generated/openapi/typescript-axios/ --reserved-words-mappings protected=protected",
+    "build": "npm run build-ts && npm run build:dev:backend:postbuild",
+    "build-ts": "tsc",
+    "build:dev:backend:postbuild": "npm run copy-yarn-lock",
+    "watch": "npm-watch",
+    "webpack": "npm-run-all webpack:dev",
+    "webpack:dev": "npm-run-all webpack:dev:node webpack:dev:web",
+    "webpack:dev:node": "webpack --env=dev --target=node --config ../../webpack.config.js",
+    "webpack:dev:web": "webpack --env=dev --target=web --config ../../webpack.config.js",
+    "copy-yarn-lock": "cp -af ../../yarn.lock ./dist/yarn.lock"
+  },
+  "dependencies": {
+    "@hyperledger/cactus-common": "1.1.3",
+    "@hyperledger/cactus-core": "1.1.3",
+    "@hyperledger/cactus-core-api": "1.1.3",
+    "fabric-network": "1.4.19",
```

Comment:
@barnapa Need to update the versions here, please see details above.

Reply:
@petermetz Done, please double check :)